replication-api-impl/src/main/java/org/codice/ditto/replication/api/impl/SyncHelper.java

```diff
@@ -201,7 +197,7 @@ private Filter buildFilter() {
     final List<Filter> failureFilters = createFailedMetacardFilters();
     final ReplicationStatus lastSuccessfulRun =
         history
-            .getReplicationEvents(config.getName())
+            .getByReplicatorId(config.getName())
```

Comment:
❗️ ❓ Should we not be cross-referencing objects using their internal unmodifiable ids as opposed to names specified by users? Names can be changed, not ids.